### PR TITLE
Amazing project

### PR DESCRIPTION
**Pull request by penalosa**
I’d like to give this project the recognition it deserves, as an amazing project.

Also please give me a free t shirt thx

### CODE REVIEW

**Comment by smartclash** [comment]
Why'd you folks spam the repository with meaningless PRs?

**Comment by pkage** [comment]
sam i swear to god

**Review by pkage — CHANGES_REQUESTED** [review]
no

**Comment by neanias** [comment]
@pkage I can't see which changes you've requested, please could you be more specific? I think this is an amazing project that truly deserves more recognition from the get-go!

**Comment by smartclash** [comment]
> @pkage I can't see which changes you've requested, please could you be more specific? I think this is an amazing project that truly deserves more recognition from the get-go!

LOL, please let this be a joke xD

**Review by findoslice — COMMENTED** [review]
I've added myself as a reviewer just so I can say "no" with a little more emphasis

**Review by penalosa — COMMENTED** [review]
Yes

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
-# SIG: Web - website2017 - an amazing project
-This is [CompSoc Edinburgh](https://comp-soc.com)'s website, built by SIG: Web.
+# SIG: Web - website2020 (up to date!) - a super duper amazing project
```
Comment (pkage) [inline README.md:1]: actually shit i didn't realize this was actually very out of date

File: README.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-## How to test
-It uses Jekyll. It also works nicely with Docker.
+## Running on your computer
+It uses Jekyll. It also works very amazingly nicely with Docker.
 
-If you use Linux or macOS, run `docker-run.sh`. It will automatically build, server, and watch the directory for changes. It will listen on port 4000.
+If you use Linux or macOS, run `docker-run.sh`. It will automatically build, server, and watch the directory for changes and it will listen on port 4000. 
 
-If you're on Windows, please contribute some instructions. You'll hopefully be able to read `docker-run.sh` and adapt the commands to play nice with docker-machine, or native Docker if that's your thing.
+If you're on Windows (I'm sorry), please contribute some instructions. You'll hopefully be able to read `docker-run.sh` and adapt the commands to play nice with docker-machine, or native Docker if that's your thing.
```
Comment (pkage) [inline README.md:9]: maybe we can offer to send flowers or chocolates

File: README.md [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
 
-### Running without Docker
+## Running on your computer without Docker
```
Comment (pkage) [inline README.md:11]: good clarification

**Comment by pkage** [comment]
yknow i think this is getting ready to merge

File: README.md [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
 If you have `bundler` installed, type:
 
 - `bundle`, to install the Gemfile packages
 - `bundle exec jekyll serve` to host a local server that will watch the directory for changes
+
+Thanks for visiting!!
```
Comment (neanias) [inline README.md:17]: Sorry, but this double exclamation mark clearly breaches CompSoc's strategic exclamation mark reserve. Please reduce this to a single bang or you shall be held accountable.

**Review by neanias — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (findoslice) [reply README.md:1]: Can we make this just "website" as opposed to including 2020? is there a need for the year?

File: README.md (hunk 2 above)
Comment (findoslice) [reply README.md:9]: Does this maybe read a bit better?
```suggestion
If you're on Windows, reconsider your life decisions.
```

File: README.md (hunk 4 above)
Comment (findoslice) [reply README.md:17]: @neanias I'll reduce you to a single bang

Comment (neanias) [reply README.md:17]: @findoslice I thought you'd never asl

File: README.md (hunk 1 above)
Comment (penalosa) [reply README.md:1]: I think it's important to memorialise0 the great year that is 2020

File: README.md (hunk 3 above)
Comment (penalosa) [reply README.md:11]: Thx

**Review by qaisjp — APPROVED** [review]
Lgtm shipit

**Comment by penalosa** [comment]
plz merge @compsoc-edinburgh/sigweb

**Comment by penalosa** [comment]
fun was had by all, but i think it's time for this to be closed :(

**Pull request closed without merging**